src/blog/2018-12-24/anouncing-stryker-0-34.pug

```diff
@@ -0,0 +1,128 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        Install Stryker via the command line:
+        
+        ```shell
+        npm i stryker stryker-api --save-dev
+        ```
+
+        Or run `stryker init` in the root of your project:
+
+        ```shell
+        npx stryker init
+        ```
+        
+        Some features we want to talk about:
+
+        * [Command test runner](#command-test-runner)
+        * [Zero configuration](#zero-configuration)
+        * [Init presets](#init-presets)
+        * [Web component tester test runner plugin](#web-component-tester-test-runner-plugin)
+
+        ## Command test runner
+
+        Stryker always was and always will be test runner agnostic. It doesn't care which test runner you use.
+        However, it always required you to install a test runner plugin. 
+        Are you using Jest? Install the 'stryker-jest-runner'. Want to use mocha? Then the 'stryker-mocha-runner' is just what you need.
+        This works great if the test runner of choice is supported, but what do you do when your test runner isn't supported?
+        Like [Ava.js](https://github.com/stryker-mutator/stryker/issues/243) or [node-tap](https://github.com/stryker-mutator/stryker/issues/325)
+        or [Intern](https://github.com/stryker-mutator/stryker/issues/430) or 
+        maybe you have a custom `npm test` command. This is why we've added the `'command'` test runner.
+
+        Configure it in your stryker configuration:
+
+        ```js
+        // stryker.conf.js
+        {
+            testRunner: 'command',
+            // Or leave out testRunner, 'command' is the default test runner
+            commandRunner: { 
+                command: 'npm run mocha' // optionally choose a different command to run
+            }
+        }
+        ```
+
+        Stryker ships with the command test runner, so no need for additional plugins. The command test runner will simply run your `npm test` script. 
+        If the exit code is `0`, the tests succeeded. If not, they failed. Stryker is limited to optimize for performance, so if your test runner is supported via a test runner plugin, that would still be the way to go.
```

Comment:
What do you mean with 'Stryker is limited to optimize for performance'?

Reply:
I've changed it to:

>Stryker is limited in the ways it can optimize for performance. For example, it starts a fresh node process for each test run.
> So if your test runner is supported via a test runner plugin, that would still be the way to go.

Is this better?